packages/client/package.json

```diff
@@ -60,6 +60,7 @@
     "react-error-boundary": "5.0.0",
     "react-hook-form": "7.54.2",
     "react-router-dom": "7.1.5",
+    "strip-indent": "4.0.0",
```

Comment:
That leftovers right? should be removed

Reply:
Weird. Thought I deleted that. Thanks

Reply:
Ah, it's removed in another PR. Wasn't related to this

Reply:
done